go-controller/pkg/ovn/common.go

```diff
@@ -21,6 +21,32 @@ func hashedAddressSet(s string) string {
 	return fmt.Sprintf("a%s", hashString)
 }
 
+func (oc *Controller) forEachAddressSet(iteratorFn func(string, string,
```

Comment:
I think the function name is better as "forEachAddressSetUnhashedName"

Reply:
Done